packages/wrangler/src/dev.tsx

```diff
@@ -426,22 +441,38 @@ export async function startDev(args: StartDevOptions) {
 		const getLocalPort = memoizeGetPort(DEFAULT_LOCAL_PORT);
 		const getInspectorPort = memoizeGetPort(DEFAULT_INSPECTOR_PORT);
 
+		const cliVars =
+			args.var?.reduce<Record<string, string>>((collectVars, v) => {
+				const [key, ...value] = v.split(":");
+				collectVars[key] = value.join("");
+				// todo: detect dupes and throw?
```

Comment:
Could do this with a `new Set` or `new Map` original length > Set then `throw`

Reply:
Thinking now I won't do the deduping at all, might be overkill, can revisit later.